### PR TITLE
// Pull Request template proposal

### PR DESCRIPTION
**Pull request by xBorderie**
## Description

This PR adds a PR template to the repository, [as made possible by recent updates to GitHub](https://github.com/blog/2111-issue-and-pull-request-templates). We hope this will help the community create better PR!

More info about GitHub PR templates: https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

Note that this creates an invisible `.github/` directory at the root of project (see last paragraph of first link).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Description
+
+<!--
+A few sentences describing the overall goals of the pull request's commit(s). 
+Explain the motivation for making this change. What existing problem does the pull request solve? Why is it the best way to solve it? Give us as much details as possible: you want to convince the PS that your PR must be merged :)
+Describe any possible limitations, side effects, etc.
+Reference any other pull requests that might be related (for instance, if you are rebasing a PR from another branch).
+-->
+ 
+This PR is a (BUG FIX / NEW FEATURE  / TEXT CHANGE). <!-- Delete the unneeded mentions and add your long description right below. -->
```
Comment (mickaelandrieu) [inline .github/PULL_REQUEST_TEMPLATE.md]: You need to put your comment a line below in order to be correctly parser by Github (see the preview).

Or maybe is that intended ?

**Comment by mickaelandrieu** [comment]
Nice contribution !

I'm not sure of the interest of commented markdown cause contributors won't never see this content.
In your opinion, how many time of contributor attention do we have ? 30s, 1min, minutes ?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (xBorderie) [inline .github/PULL_REQUEST_TEMPLATE.md]: My mistake! Fixed, thanks!

**Comment by xBorderie** [comment]
I agree that the whole intro is overly long for now, but I want to iterate on this with the first few PR. I do hope that see this block of text will push contributors to pay better attention to the way they fill their PR description field. We'll see with the first results!

**Comment by kpodemski** [comment]
@xBorderie you should add information about `BO:` `FO:` `MO:` `CORE:` here, people will save some time instead of going to yet another website to read the docs

**Comment by xBorderie** [comment]
@kpodemski I'm not sure, there's a already a lot of content in the header. I know that they are unlikely to click a link, but I'm afraid there's already a lot in there. Do you have a suggestion?

**Comment by kpodemski** [comment]
One simple paragraph:
### About commits name
Commit name has three parts:
`[-]` + `FO :` + `Your commit name for things related to Front-Office`

`[-]` means that commit is about bug fix, it can be also:
`[*]` for improvement and:
`[+]` for new feature

`FO :` means that commit is related to Front-Office, it can be also:
`BO :` for back-office,
`MO :` for modules,
`CORE :` for Core

last part is your commit regular name.

**Comment by xBorderie** [comment]
Ah, was adding your idea and simplifying the intro while you were typing your suggestion. What do you think!
You're right about //, I'll add it -- and in the doc page as well.

**Comment by kpodemski** [comment]
@xBorderie it's good :) you can also add MO : and information that it is related to modules repositories

**Comment by kpodemski** [comment]
:+1:

**Pull request merged**